### PR TITLE
feat: Add Custom JS

### PR DESCRIPTION
**Pull request by WmeLuna**
Adds an option to add javascript to be inserted into the webview

### CODE REVIEW

File: src/settings-tab.ts [hunk 1]
```diff
@@ -149,6 +149,22 @@ export class CustomFramesSettingTab extends PluginSettingTab {
                         await this.plugin.saveSettings();
                     });
                 });
+                new Setting(content)
+                .setName("Additional JS")
+                .setDesc(createFragment(f => {
+                    f.createSpan({ text: "A snippet of additional JS that should be applied to this frame." });
```
Comment (Ellpeck) [inline src/settings-tab.ts:155]: I think it would be more user-friendly to use the full word "JavaScript" here.

File: src/settings-tab.ts [hunk 2]
```diff
@@ -133,7 +133,7 @@ export class CustomFramesSettingTab extends PluginSettingTab {
                         await this.plugin.saveSettings();
                     });
                 });
-            new Setting(content)
+                new Setting(content)
```
Comment (Ellpeck) [inline src/settings-tab.ts:136]: nbd but you're doing weird indentation here

**Review by Ellpeck — CHANGES_REQUESTED** [review]
Thanks so much for this PR!! I requested a few minor changes, but everything else looks great.

File: src/settings-tab.ts (hunk 1 above)
Comment (WmeLuna) [reply src/settings-tab.ts]: Fixed!

File: src/settings-tab.ts (hunk 2 above)
Comment (WmeLuna) [reply src/settings-tab.ts]: ngl not sure what happened there but fixed

**Comment by Ellpeck** [comment]
Thanks so much! Looks great.

**Comment by WmeLuna** [comment]
I had forgot to put `customJs` in [`src/settings-tab.ts#L201`](https://github.com/WmeLuna/ObsidianCustomFrames/commit/ae77dd57e41ca779e6e8ae40e2a7a96e79fb1963)
So thats fixed in my the commit linked.
For some reason though the node script `dev` can build it without that commit with no issues, but not the script `build`. plugin works fine with or without it tho :P

**Comment by unix2dos** [comment]
can this mr be released?

**Pull request merged**